stan/math/prim/fun/cholesky_factor_constrain.hpp

```diff
@@ -64,25 +64,24 @@ Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> cholesky_factor_constrain(
  *
  * @tparam T type of elements in the matrix
```

Comment:
Pretty sure you can copy pasta from the other comments for all of these

Reply:
done